content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 49
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on
+and use the web-based graphical user interface for viewing service graphs of
+the mesh as well as your Istio configuration objects. You will also use the
```

Comment:
for viewing service graphs of the mesh and your Istio configuration objects.

Reply:
this is done.